packages/pipeline-editor/src/PipelineSubmissionDialog.tsx

```diff
@@ -88,22 +91,26 @@ export class PipelineSubmissionDialog extends React.Component<IProps, IState> {
         />
         <br />
         <br />
-        <label htmlFor="runtime_platform">Runtime Platform:</label>
-        <br />
-        <select
-          id="runtime_platform"
-          name="runtime_platform"
-          className="elyra-form-runtime-platform"
-          data-form-required
-          defaultValue={selectedRuntimePlatform}
-          onChange={this.handleUpdate}
-        >
-          {validSchemas.map(schema => (
-            <option key={schema.name} value={schema.name}>
-              {schema.display_name}
-            </option>
-          ))}
-        </select>
+        {!this.props.runtime && (
+          <div>
+            <label htmlFor="runtime_platform">Runtime Platform:</label>
+            <br />
+            <select
+              id="runtime_platform"
+              name="runtime_platform"
+              className="elyra-form-runtime-platform"
+              data-form-required
+              defaultValue={selectedRuntimePlatform}
+              onChange={this.handleUpdate}
```

Comment:
if we use `onChange` to update the `selectedRuntimePlatform`, then `defaultValue` should just be `value`